### ISSUE TITLE
Uningest tabular files

### ISSUE THREAD

**Issue by DS-INRA** [IQSS/dataverse-frontend#308]
As a super-admin (end goal would be **curator**, but safer for experimenting first),
I want to be able to uningest a .tab file from the UI,
to provide faster support to dataset owner whose file is not correctly ingested or should not be ingested.

Adding an option in the Edit File Dropdown.
In the file page :
![image](https://github.com/IQSS/dataverse-frontend/assets/46443753/795b83fd-63d5-4f05-a8fc-6d73f39a9225)
In the dataset page's file tab :
![image](https://github.com/IQSS/dataverse-frontend/assets/46443753/d3abd148-b726-430d-9627-4951b0a05ab9)

A confirmation window with Continue/Cancel should then be displayed in order to confirm the uningestion and avoid missclicking errors.

Suggestion of text (in `text-warning` format) : 
This file will be uningested, you will lose of the vantages of the ingestion. Click Continue to do so.
For more information about ingestion and uningestion, check out the [Tabular Data File Ingest Guide](https://guides.dataverse.org/en/latest/user/tabulardataingest/index.html)

**Comment by qqmyers** [comment]
FWIW: QDR has implemented this - Uningest and Reingest. I forget what permission is needed, but since it is reversible, I don't think it was too strict. We did this before the messaging changes, when a failed ingest sounded more like an error. If there's interest now and you haven't already done the work, I could probably make a PR.

**Comment by DS-INRA** [comment]
Great, there is definitively interest and we haven't worked on it yet so it would be more than welcome !

**Comment by qqmyers** [comment]
@DS-INRA - Ah sorry - to clarify, QDR did this on the current UI and that's the PR I could create. We haven't worked in the SPA yet. I didn't look to see which repo this was in.

**Comment by DS-INRA** [comment]
Yes I expected it was the case, so still very useful. Do you want me to create the issue on the other side for it ?

**Comment by qqmyers** [comment]
Yeah - that would be useful.
